package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A node wrapper for the HubSpot API",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run mocha",
+    "test": "npm run lint && npm run mocha && npm run tsc",
+    "tsc": "tsc",
+    "ts-node": "ts-node test/typescript/hubspot.ts",
```

Comment:
This does a runtime check. Hence, I didn't include it in the test command. Perhaps this could eventually be moved to something only the CI calls?